.travis.yml

```diff
@@ -98,31 +98,30 @@ jobs:
         - skip
     - stage: conda-build
       install:
-        - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
-        - bash miniconda.sh -b -p $HOME/miniconda
-        - export PATH="$HOME/miniconda/bin:$PATH"
+        - cd ../
+        - bash ./tethys/scripts/install_tethys.sh --partial-tethys-install me -n tethys -s $PWD/tethys
+        - . ~/miniconda/etc/profile.d/conda.sh
+        - conda activate tethys
         - hash -r
         - conda config --set always_yes yes --set changeps1 no
         - conda update -q conda
         - conda install conda-build anaconda-client
-        - conda config --add channels bokeh/label/dev
-        - conda config --add channels conda-forge
-        - conda config --add channels tethysplatform/label/dev
-        - conda config --add channels tethysplatform
```

Comment:
It looks like the conda build isn't working and I think it is because you've removed these. I agree with removing these, but then we need to specify the channels to use during the build to get all of the dependencies. I'll add a suggestion for that.